src/perftest/java/com/lmax/disruptor/support/MultiBufferBatchEventProcessor.java

```diff
@@ -57,8 +57,6 @@ public void run()
         }
 
         final int barrierLength = barriers.length;
-        final long[] lastConsumed = new long[barrierLength];
-        fill(lastConsumed, -1L);
```

Comment:
What role does the lastConsumed array serve.  It seems to only be declared and used here no where else.  It appears to be redundant.